config/lib/seed.js

```diff
@@ -4,6 +4,11 @@ var mongoose = require('mongoose'),
   chalk = require('chalk'),
   crypto = require('crypto');
 
+// set default seed options
+var seedOptions = {
+  logResults: true // log the results of the seeding to the console
```

Comment:
@codydaig What if we set the default here, from the env config? While keeping the seedOptions object.